src/ZombieDriver.php

```diff
@@ -869,7 +869,7 @@ protected function triggerBrowserEvent($event, $xpath)
 JS;
         $out = $this->server->evalJS($js);
         if (!empty($out)) {
-            throw new DriverException(sprintf("Error while processing event '%s'", $event));
+            throw new DriverException(sprintf("Error while processing event '%s': $out", $event));
```

Comment:
please use a second placeholder in the sprintf instead of doing string interpolation (which could break the sprintf if there is a ``%`` in ``$out``)